### ISSUE TITLE
SC "level" label doesn't follow the reading order

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag#1428]
**expected**: the level is included immediately after the title of the SC, like this:

> Success Criterion 1.1.1 Non-text Content §
> (Level A)
> Understanding Non-text Content|
> How to Meet Non-text Content

**Actual**: The reading order is like this:

> Success Criterion 1.1.1 Non-text Content§
> Understanding Non-text Content|
> How to Meet Non-text Content(Level A)

There are also two missing spaces, one between the title and the paragraph link, and the second between the How To Meet link, and the level.

Arguably, this is a failure of 1.3.2 Meaningful Sequence. This problem was introduced in WCAG 2.1. It'd be great if we could fix this in 2.2.

**Comment by mbgower** [comment]
@michael-n-cooper At least in the rendered html, this looks easy to fix. The level information is set in a paragraph that follows the div containing the links to the Understanding and How to meet documents.
Putting the paragarph in front of this div resolves the reading order, yet does not appear to affect the visual presentation.

Is that difficult to implement? The following illustration just show a screen snippet of the change made inside a browser via the chrome html editor.

![image](https://user-images.githubusercontent.com/14298245/140831792-fe22073b-c5b6-4849-b33b-bea98d032502.png)

<img width="1239" alt="Screen Shot 2021-11-08 at 2 59 41 PM" src="https://user-images.githubusercontent.com/14298245/140831828-469ab231-596a-4673-be28-40df89a240ab.png">
